drivers/wifi/winc1500/bsp/include/nm_bsp.h

```diff
@@ -0,0 +1,284 @@
+/**
+ *
+ * \file
+ *
+ * \brief WINC BSP API Declarations.
+ *
+ * Copyright (c) 2016 Atmel Corporation. All rights reserved.
```

Comment:
Sounds like this file should go somewhere in ext/  or?

What goes into all directories but ext, is made for zephyr so there is no such thing as redefining stuff like NULL, uints ... etc... that's useless: Zephyr provides all necessary bits and pieces.